DigitalLearningSolutions.Data.Tests/Services/UnlockServiceTests.cs

```diff
@@ -1,149 +0,0 @@
-﻿namespace DigitalLearningSolutions.Data.Tests.Services
-{
-    using System.Threading.Tasks;
-    using DigitalLearningSolutions.Data.DataServices;
-    using DigitalLearningSolutions.Data.Models;
-    using DigitalLearningSolutions.Data.Models.Email;
-    using DigitalLearningSolutions.Data.Services;
-    using DigitalLearningSolutions.Data.Tests.TestHelpers;
-    using FakeItEasy;
-    using Microsoft.Extensions.Configuration;
-    using Microsoft.FeatureManagement;
-    using NUnit.Framework;
-
-    public class UnlockServiceTests
-    {
-        private IConfiguration configuration = null!;
-        private IEmailService emailService = null!;
-        private IFeatureManager featureManager = null!;
-        private INotificationDataService notificationDataService = null!;
-        private NotificationService notificationService = null!;
-        private IUserService userService = null!;
-
-        [SetUp]
-        public void Setup()
-        {
-            configuration = A.Fake<IConfiguration>();
-            notificationDataService = A.Fake<INotificationDataService>();
-            emailService = A.Fake<IEmailService>();
-            featureManager = A.Fake<IFeatureManager>();
-            userService = A.Fake<IUserService>();
-
-            A.CallTo(() => notificationDataService.GetUnlockData(A<int>._)).Returns(
-                new UnlockData
-                {
-                    ContactEmail = "recipient@example.com",
-                    ContactForename = "Forename",
-                    CourseName = "Activity Name",
-                    CustomisationId = 22,
-                    DelegateId = 1,
-                }
-            );

```

Comment:
We seem to be missing this set up, but it looks like everything works fine without it. So I guess it is just useless.

Reply:
None of the tests actually tested that any of these values were part of the email, so it didn't actually do anything.